torch/_inductor/kernel/templated_attention.py

```diff
@@ -49,10 +49,8 @@ def sdpa_grid(batch_size, num_heads, num_queries, d_model, meta):
     H = {{size("Q", 1)}}
     N_CTX = {{size("Q", 2)}}
 
-    # TODO I think we should do some performance work
-    # to find the optimal calls for perf/accuracy to tl.dot
     qk_scale = 1.0
-    MATMUL_PRECISION = tl.float16
+    MATMUL_PRECISION = tl.float32
```

Comment:
this upcasts the bfloat16 /float16 case ?

Reply:
yeah I was trying to get float32 to work, gimme a bit.